packages/@coorpacademy-components/src/molecule/learner-skill-card/test/fixtures/no-skill-courses.js

```diff
@@ -1,11 +0,0 @@
-import {defaultProps} from './default';
```

Comment:
peut être garder celui ci, 
le renommer en "no-content",
même si je vois pas trop le scénario mais on a une condition si content => display progressBar,
donc ce serait une fixture pour tester ce use case qui n'affiche pas la progressBar

Reply:
@adamska27 en vrai cette use case là on l'a déjà c'est la fixture "no skill courses and questions"
![Capture d’écran 2025-01-03 à 16 00 13](https://github.com/user-attachments/assets/0752d8e3-f44f-4730-b9eb-f32b12f3a71c)

Reply:
mais vu qu'il n'y a plus de skill courses je peux la renommer en "no content"

Reply:
ah oui, on a les 2 sur celui là donc ça va c'et bon pas la peine d'en rajouter 👍